build.sh

```diff
@@ -13,4 +13,4 @@ while [[ -h $source ]]; do
 done
 
 scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
-"$scriptroot/eng/build.sh" --restore --build --build-native $@
+"$scriptroot/eng/build.sh" --restore --build --native $@
```

Comment:
The change was because as far as I can tell ps1 scripts can't have dashes in the option name. 

Yes, I guess always building managed and native and then having "--nonative" or "--nomanaged" options does make more sense.

Reply:
To be consisten with the other repos, `skipnative` and `skipmanaged`

Reply:
done.